canvas_handler.py

```diff
@@ -162,22 +152,20 @@ def untrack_course(self, course_ids_str: Tuple[str, ...]):
         """
 
         course_ids = self._ids_converter(course_ids_str)
+        c_ids = {c.id for c in self.courses}
```

Comment:
Sets are much faster for checking the contents. I did this a few times in canvas_handler.